build.gradle

```diff
@@ -2,10 +2,8 @@ buildscript {
     apply from: rootProject.file('dependencies.gradle')
     repositories {
         jcenter()
-        google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
```

Comment:
Since plugin/compiler/lib does not apply android plugin, this is not needed. I guess it was a copy paste.